frontend/packages/knative-plugin/integration-tests/features/serverless-functions/serverless-function.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Visualisation of serverless fuctions
+              As a user, I should be able to differentiate between a plain Knative Service and a Serverless Function when looking at the Details page of the KSVC and the details tab of the side panel in Topology
+
+        Background:
+            Given user is at developer perspective
+              And user has installed Serverless Operator
+              And And user has created knativeServing instance in knative-serving namespace
+              And user has created or selected namespace "aut-serverless-function"
+
+
+        @regression, @manual
+        Scenario: Visualize serverless function in topology chart view
+            Given user has created serverless function "nodetest"
+              And user is at Topology list page
+             When user clicks on Topology chart view
+             Then user will be see name of the serverless function as fx label followed by KSVC label and name "nodetest"
+              And user will not see git and build decorator associated with the function
+
+
+        @regression, @manual
+        Scenario: Visualize serverless function in topology list view
+            Given user has created serverless function "nodetest"
+              And user is at Topology chart page
+             When user clicks on Topology list view
+             Then user will be see name of the serverless function as KSVC label followed by fx label and name "nodetest"
+
+
+        @regression
+        Scenario: Sidebar of serverless function
+            Given user has created serverless function "nodetest"
+              And user is at Topology chart page
+             When user clicks on "nodetest"
+             Then user will be see type as "function" mentoned in the Details tab of sidebar
```

Comment:
two things here:
- The type header has value "Function". Is there a reason title-case is not retained?
- typo "mentoned" --> "mentioned"

Reply:
Updated